modules/progress/js/progress.js

```diff
@@ -32,8 +32,8 @@
           $.noop
         );
         progressBar.setProgress(67, 'Progress message', 'Progress label');
-        var element = $(progressBar.element).addClass(
-          'ajax-progress ajax-progress-bar'
+        var element = $(
+          Drupal.theme('ajaxProgressBar', progressBar.element),
```

Comment:
https://www.drupal.org/project/drupal/issues/3059847 won't be added before 8.8.0, I suggest to use here a condition. We have to check that the `ajaxProgressBar` theme function exists before using it.